### PR TITLE
Update readme documentation

### LINKED ISSUE #67: Update repository documentation
### Describe the solution you'd like

Is required to improve all the project documentation before the next releases and to help people who want to contribute by providing clear information about the project.

### Additional context

To solve this issue is required:

- [x] Improve the project description.
- [x] Add information about the project **roadmap**.

### PR DESCRIPTION
**Pull request by SanGeeky**
## Description
Closes #67 

This PR updates the project Readme with the following documentation changes:
- Update project description.
- Describes project roadmap.
- Update the project setup with their config file.

## Type of change
- [x] This change requires a documentation update

## Checklist
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by miguelnietoa — CHANGES_REQUESTED** [review]
@SanGeeky Comments added!

@CristhianRodriguezMolina could you please help me with the review of the setup instructions?

File: README.md [hunk 1]
```diff
@@ -1,28 +1,110 @@
...
+    `mix deps.get`
+
+2. Mintacoin requires a development configuration file. Copy the example with the following command:
+
+    `cp config/dev.exs.example config/dev.exs`
+
+3. Within the file `dev.exs` replace the following variables
+
+    - **`"STELLAR_FUND_SECRET_KEY"`** with the secret key created from the [Stellar Laboratory][stellar-laboratory].
+    - **`"BLOCKCHAINS_NETWORK"`** with `"testnet"`.
```
[the remark this answers is not in the document]
Comment (CristhianRodriguezMolina) [reply README.md]: This is used to get the blockchains matching with the `network` field of the blockchain table, this field is an `:atom` so we can set this here as an atom.

[the remark this answers is not in the document]
Comment (CristhianRodriguezMolina) [reply README.md]: This is related to the `network` field of the blockchain table which is an atom, so we can set this as an atom, what do you think @SanGeeky?

[the remark this answers is not in the document]
Comment (SanGeeky) [reply README.md]: After reviewing for the development process we will define this variable as atom.

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]
@SanGeeky looks really good! Just one minor improvement ... It is important to clarify the blockchain network we're using for the API.

**Review by CristhianRodriguezMolina — DISMISSED** [review]

**Pull request merged**